html-css/week1/Nina/portfolio/style.css

```
@@ -0,0 +1,376 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+
+  font-family: "Open Sans", sans-serif;
+  font-style: normal;
+  display: flex;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+  color: black
+}
+
+a:hover {
+
+  text-decoration: underline;
+
+}
+
+.container {
+  max-width: 1440px;
+  margin: auto;
+  gap: 30px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+
+  @media(max-width:1440px) {
+    padding-left: 30px;
+    padding-right: 30px;
+  }
+}
+
+nav {
+  max-width: 1440px;
+  width: 100%;
+  background-color: rgba(212, 236, 255, 0.623);
+  padding: 10px;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+
+  ul {
+    display: flex;
+    align-items: Center;
+    list-style-image: none;
+
+
+    li {
+      display: inline-block;
+      margin-right: 10px;
+
+      a {
+        display: inline-block;
+        padding: 10px 20px;
+
+        &:hover {
+          background-color: rgba(241, 240, 240, 0.962);
+          border-radius: 10px;
+          text-decoration: none;
+        }
+
+      }
+    }
+  }
+
+  @media(max-width:768px) {
+    nav {
+      width: 100%;
+      margin-top: 10px;
+    }
+
+    .contact-me-button {
+      height: auto;
+    }
+  }
+}
+
+.button {
+  background-color: #0a61ae;
+  border-radius: 10px;
+  color: white;
+  height: 40px;
+
+  &:hover {
```

You can add `:focus-visible` to apply the same styling on accessing the element on tab key, too.
```suggestion
  &:hover,
  &:focus-visible {
```